projects/robots/boston_dynamics/spot/controllers/spot_moving_demo/spot_moving_demo.c

```diff
@@ -0,0 +1,418 @@
+  movement_decomposition(motors_target_pos_2, DURATION);
+  movement_decomposition(motors_target_pos_3, DURATION);
+  movement_decomposition(motors_target_pos_4, DURATION);
+  movement_decomposition(motors_target_pos_5, DURATION);
+  movement_decomposition(motors_target_pos_6, DURATION);
+  movement_decomposition(motors_target_pos_7, DURATION);
+  movement_decomposition(motors_target_pos_8, DURATION);
+  movement_decomposition(motors_target_pos_9, DURATION);
+  movement_decomposition(motors_target_pos_10, DURATION);
+  movement_decomposition(motors_target_pos_11, DURATION);
+  movement_decomposition(motors_target_pos_12, DURATION);
+  movement_decomposition(motors_target_pos_13, DURATION);
+  lie_down();
+  passive_wait(1);
+  stand_up();
+}
+
+static void check_keyboard() {
+  const int key = wb_keyboard_get_key();
+  if ((key >= 0) && key != old_key) {
+    switch (key) {
+      case WB_KEYBOARD_UP:
+        printf("Robot stands up\n");
+        stand_up();
+        autopilot = false;
+        break;
+      case WB_KEYBOARD_DOWN:
+        printf("Robot lies down\n");
+        lie_down();
+        autopilot = false;
+        break;
+      case WB_KEYBOARD_END:
+      case 'L':
+        if (key != old_key)  // perform this action just once
+          L_pressed = !L_pressed;
+        break;
+      case 'D':
+        if (key != old_key)
+          demo = !demo;
+        break;
+      case 'C':
+          if (key != old_key)
+            C_pressed = !C_pressed;
+          break;
+      case 'A':
+        if (key != old_key)
+          autopilot = !autopilot;
+        break;
+      default:
+        fprintf(stderr, "Wrong keyboard input\n");
+        break;
+    }
+  }
+  if (autopilot != old_autopilot) {
+    old_autopilot = autopilot;
+    if (autopilot)
+      printf("Auto control\n");
+    else
+      printf("Manual control\n");
+  }
+  if (C_pressed != old_C_pressed) {
+    old_C_pressed = C_pressed;
+    if (cameras_enabled) {  // Switch off
+      for (int i = 0; i < NUMBER_OF_CAMERAS; i++)
+        wb_camera_disable(cameras[i]);
+      cameras_enabled = false;
+      printf("Cameras OFF\n");
+    } else {
+      for (int i = 0; i < NUMBER_OF_CAMERAS; i++)
+        wb_camera_enable(cameras[i], TIME_STEP);
+      cameras_enabled = true;
+      printf("Cameras ON\n");
+    }
+  }
+  if (L_pressed != old_L_pressed) {
+    old_L_pressed = L_pressed;
+    if (leds_enabled) {  // Switch off
+      for (int i = 0; i < NUMBER_OF_LEDS; i++)
+        wb_led_set(leds[i], LEDS_OFF);
+      leds_enabled = false;
+      printf("LEDs OFF\n");
+    } else {
+      for (int i = 0; i < NUMBER_OF_LEDS; i++)
+        wb_led_set(leds[i], LEDS_ON);
+      leds_enabled = true;
+      printf("LEDs ON\n");
+    }
+  }
+  old_key = key;
+}
+
+// Demonstration
+static void run_demo() {
+  printf("The demonstration will start in...\n");
+  printf("3 \n"); passive_wait(1);
+  printf("2 \n"); passive_wait(1);
+  printf("1 \n"); passive_wait(1);
```

Comment:
```suggestion
  printf("3 \n");
  passive_wait(1);
  printf("2 \n");
  passive_wait(1);
  printf("1 \n");
  passive_wait(1);
```
This is not accepted by clang-format.